packages/ui/__tests__/components/atoms/Switch.test.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2023 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import {shallow} from 'enzyme';
+import {Switch as RNSwitch} from 'react-native';
+import {Switch} from '@axelor/aos-mobile-ui';
+
+describe('Switch Component', () => {
+  it('renders without crashing', () => {
+    const wrapper = shallow(<Switch />);
+    expect(wrapper.exists()).toBe(true);
+  });
+
+  it('renders the switch with the correct initial value', () => {
+    const props = {
+      isEnabled: true,
+      handleToggle: jest.fn(),
+    };
+    const wrapper = shallow(<Switch {...props} />);
+    expect(wrapper.prop('value')).toBe(true);
+  });
+
+  it('should call handleToggle function with value true when switching on', () => {
+    const handleToggle = jest.fn();
+    const props = {
+      isEnabled: false,
+      handleToggle,
+    };
```

Comment:
Please simplify props creation, it is always the same thing. Define handleToggle function at the beginning and give isEnabled directly to component as it's the only props that is modified